examples/resources/gitlab_group_saml_link/resource.tf

```
@@ -0,0 +1,5 @@
+resource "gitlab_group_saml_link" "test" {
+  group_id        = "12345"
+  access_level    = "Developer"
```

@timofurrer  I think we should make the example lowercase, since we explicitly have the `stringInSlice` function set to case sensitive:

```suggestion
  access_level    = "developer"
```